modules/instrument_builder/test/TestPlan.md

```diff
@@ -1,41 +1,35 @@
-    1.d Add Textbox
-      1.d.1 Validate that this requires both QuestionName and QuestionText
-    1.e Add Textarea
-      1.e.1 Validate that this requires both QuestionName and QuestionText
-      1.e.2 Validate that not_answered option is added in the linst file for this question
-    1.f Add Dropdown
-     1.f.1 Validate that this requires both QuestionName and QuestionText
-     1.f.2 Validate not_answered option is added as last dropdown option in the linst file
-           for this question
-    1.g Add Multiselect
-      1.g.1 Validate that this requires both QuestionName and QuestionText
-      1.g.2 Validate not_answered option is added as last dropdown option in the linst file
-            for this question
-    1.h Add Date
-      1.h.1 Validate that this requires both QuestionName and QuestionText
-      1.h.2 Validate not_answered option is added as last dropdown option in the linst file
-            for this question
-      1.h.3 Validate adding range (startyear-endyear)
-    1.i Add Numeric
-      1.i.1 Validate that this requires both QuestionName and QuestionText
-      1.i.2 Validate adding range
-    1.j Add Blank Line
-    1.k Add Page break
-2.  Edit DatabaseName and Question in the table directly and check if it sticks
-3.  Preview Instrument
-4.  Add Rules
-5.  Save Rules file
-6.  Validate Instrument (Does not save if there is a same QuestionName twice)
-7.  Save Instrument file
-8.  Re-load existing instrument
-    8.a  Add questions
-    8.b Save instrument file and ensure new changes are reflected in the file
-9. Install and test instrument and rules implemented
-10. Test functionality on mutiple broswers.
+  * 1.a Add Header 
+  * 1.b Add Label 
+  * 1.c Add Scored Field 
+    * 1.c.1 Validate that this requires both QuestionName and QuestionText 
+  * 1.d Add Textbox 
+    * 1.d.1 Validate that this requires both QuestionName and QuestionText 
+  * 1.e Add Textarea  
+    * 1.e.1 Validate that this requires both QuestionName and QuestionText 
+  * 1.f Add Dropdown   
+    * 1.f.1 Validate that this requires both QuestionName and QuestionText 
+  * 1.g Add Multiselect 
+    * 1.g.1 Validate that this requires both QuestionName and QuestionText 
+  * 1.h Add Date 
+    * 1.h.1 Validate that this requires both QuestionName and QuestionText 
+    * 1.h.2 Validate adding range (startyear-endyear) 
+  * 1.i Add Numeric 
+    * 1.i.1 Validate that this requires both QuestionName and QuestionText 
+    * 1.i.2 Validate adding range 
+  * 1.j Add Blank Line 
+  * 1.k Add Page break
+2.  Edit Question Name and Question Text in the table directly and check if it sticks.
+3.  Validate Instrument (Does not save if there is a same QuestionName twice)
+4.  Save Instrument file
+5.  Validate that not_answered option is added in the linst file for 1.e, 1.f, 1.g, 1.h
+6.  Preview Instrument [not available now].
+7.  Add Rules [not available now].
+8.  Save Rules file [not available now].
```

Comment:
Similar to the other PR, if these features don't exist yet then they shouldn't appear in the test plan. Instead a proposal should be created via Issues.

Reply:
Please ask @christinerogers for that. She requested to keep them.